src/content/architecture/route-agent/_index.en.md

```diff
@@ -8,15 +8,15 @@ responsible for setting up VxLAN tunnels and routing the cross cluster traffic
 from the node to the cluster’s active Gateway Engine which subsequently sends the
 traffic to the destination cluster.
 
-When running on the same node as the active Gateway Engine, the Route Agent creates
-a VxLAN VTEP interface to which the Route Agent instances running on the other worker
-nodes in the local cluster connect by establishing a VXLAN tunnel with the VTEP of
-the active Gateway Engine node. The MTU of VxLAN tunnel is configured based on the MTU
+When running on the same node as the active Gateway Engine, Route Agent creates
+a VxLAN VTEP interface to which Route Agent instances running on the other worker
```

Comment:
Why VxLAN? https://tools.ietf.org/html/rfc7348

Reply:
Yep, good point, I changed it to VXLAN everywhere in this file.